public/assets/css/tutorialAndTOS.css

```diff
@@ -48,41 +48,16 @@ hr{
     background-size: cover;
 }
 
-.returnbutton{
-    width: 7vh;
-    height: 7vh;
-    margin-top: 4vh;
-    margin-left: auto;
-    margin-right: auto;
-    background-color: var(--greyBtn);
-    border: 0.5vh solid black;
-    border-radius: 2vh;
-    display: flex;
-    align-items:center;
-    justify-content: center;
-}
-
-#returnbutton{
-    width: 4vh; 
-    height: 4vh;
-}
-
-.otherbutton{
-    display:none;
-    width: 7vh;
-    height: 7vh;
-}
-
 .logo{
     display:block;
     align-items: center;
     position:relative;
     text-align: center;
-   
 }
+
 #logo{
     height: 10vh;
-    margin-top: 4vh;
+    margin-top: 2vh;
     margin-left: auto;
     margin-right: auto;
 }
```

Comment:
You may rename the file without uppercases

Reply:
changes done